fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -1974,4 +1988,14 @@ public void replayAlterMaterializedViewBaseTableInfos(AlterMaterializedViewBaseT
         // recheck again
         fixRelationship();
     }
+
+    public ParseNode getDefineQueryParseNode() {
```

Comment:
is it thread-safe when accessed concurrently ?

Reply:
fixed. `defineQueryParseNode` is safe for multi threads since it is only initialized when mv becomes to active.